tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
@TravisEz13 Ok.  Once you make the change I'll update this test to use the AppVeyor credentials when available.  Thanks.